Gemfile

```diff
@@ -44,7 +44,7 @@ gem 'omniauth-saml', '~> 1.10'
 gem 'omniauth', '~> 1.9'
 
 gem 'discard', '~> 1.1'
-gem 'doorkeeper', '~> 4.2'
+gem 'doorkeeper', '~> 4.3'
```

Comment:
[Nexus Lifecycle](https://www.sonatype.com/product-nexus-lifecycle) has found that this version of `doorkeeper` violates your company's policies.

Lifecycle recommends using version [5.0.0.rc2]() instead as it does not violate any policies.